debezium-connector-postgres/src/main/java/io/debezium/connector/postgresql/RecordsStreamProducer.java

```diff
@@ -291,7 +288,8 @@ protected void generateCreateRecord(TableId tableId, Object[] rowData, BlockingC
         assert tableSchema != null;
         Object key = tableSchema.keyFromColumnData(rowData);
         Struct value = tableSchema.valueFromColumnData(rowData);
-        if (key == null || value == null) {
+        if (value == null) {
```

Comment:
In case when there is a table without primary key, but with replica identity FULL. Record will be produced, but since it has no PK it is skipped without this fix.

Reply:
Ok, got it. Thanks; so actually we were saying more in the docs than this connector actually had been doing. I'll also add a test for this.